src/components/OSCALMarkdownProse.js

```diff
@@ -8,5 +8,13 @@ import ReactMarkdown from "react-markdown";
  */
 
 export default function OSCALMarkdownProse(props) {
-  return <ReactMarkdown>{props.text}</ReactMarkdown>;
+  return (
+    <ReactMarkdown
+      components={{
+        p: React.Fragment,
+      }}
+    >
+      {props.text}
+    </ReactMarkdown>
+  );
```

Comment:
We can't do this all the time. Sometimes we need the `<p>` tags. I think we need an `OSCALMarkupLine` and `OSCALMarkupMultiline` component that handle sensible defaults and `component` limitations for each. Coordinate with @tuckerzp on how to name the compnents.

Reply:
Making separate components will work. Another approach could be to write functions in `OSCALMarkdownProse` which handle either single or multi-line markdown conversion. The default could be converting it into `<p>` tags and another function can handle conversions to React Fragments.

What is your opinion on this?

Reply:
I'd think the current `OSCALMarkupProse` would need minor changes to act as `OSCALMarkupMultiline`. Then you would also implement `OSCALMarkupLine`. I think that they could reside in the same file though.

Reply:
`OSCALMarkupLine` has been implemented and merged in